dev/doc/critical-bugs

```diff
@@ -386,6 +386,16 @@ Conversion machines
   risk: none, unless mixing open terms and primitive floats inside primitive
   arrays; critical otherwise
 
+  component: "native" conversion machine (translation to OCaml which compiles to native code)
+  summary: conversion of Prod / Prod values was comparing the wrong components
+  introduced: V8.5
+  impacted released versions: V8.5-V8.16.0
```

Comment:
_Might_ be worth clarifying that `native_compute` might not be enabled by default and point to https://coq.discourse.group/t/about-the-native-compiler-flag-and-the-coq-native-opam-package-to-be-released-before-coq-8-13-beta1/1147 or similar? I just confirmed I don't have `coq-native` installed and the example doesn't go through for me.

Reply:
Well, this is only since 8.13, before that the default was "ondemand", so the explanation might get a tad lengthy.

Reply:
"when built with native_compute enabled" is the only new info, the rest can be handled by links to existing docs.

Reply:
@Blaisorblade ok, let's do that